manifests/script-lab-prod.xml

```diff
@@ -17,7 +17,14 @@
     <AppDomain>https://github.com</AppDomain>
     <AppDomain>https://stackoverflow.com</AppDomain>
     <AppDomain>https://dev.office.com</AppDomain>
-    <AppDomain>https://script-lab-runner.azureedge.net</AppDomain>  
+    <AppDomain>https://localhost:3000</AppDomain>
```

Comment:
I am not sure we should be adding localhost to the production manifest. Office store team may have an issue with this in validation and it could be a security issue.